core/include/tangram/platform.h

```diff
@@ -14,10 +14,14 @@ using FontSourceLoader = std::function<std::vector<char>()>;
 
 struct FontSourceHandle {
     FontSourceHandle(std::string _path) : path(_path) {}
+    FontSourceHandle(std::string _path, bool _appleFont) : path(_path), appleFont(_appleFont) {}
     FontSourceHandle(FontSourceLoader _loader) : load(_loader) {}
+    FontSourceHandle() {}
 
     std::string path;
-    FontSourceLoader load;
+    FontSourceLoader load = nullptr;
+    bool appleFont = false;
```

Comment:
Can we rename this flag to `systemFontName` or `fontName` ?

Reply:
updated aptly to reflect the usage.